Source/WTF/wtf/text/ASCIILiteral.cpp

```diff
@@ -28,10 +28,16 @@
 #include <wtf/text/ASCIILiteral.h>
 
 #include <wtf/PrintStream.h>
+#include <wtf/text/MakeString.h>
 #include <wtf/text/StringView.h>
 
 namespace WTF {
 
+String ASCIILiteral::toString() const
```

Comment:
We have `String(ASCIILiteral)`. Did that not work in this case?

Reply:
It is not that. It is that when refactoring the code we were using `StringView::toString()` a lot, so when switching to `UnownedCString`, I decided to implement this to reduce changing something that could be solved my adding this small method.